packages/core/src/main/cluster/kube-auth-proxy-server.injectable.ts

```diff
@@ -24,23 +25,23 @@ const thirtySecondsInMs = 30 * 1000;
 const kubeAuthProxyServerInjectable = getInjectable({
   id: "kube-auth-proxy-server",
   instantiate: (di, cluster): KubeAuthProxyServer => {
-    const clusterUrl = new URL(cluster.apiUrl.get());
-
-    const createKubeAuthProxy = di.inject(createKubeAuthProxyInjectable);
-    const certificate = di.inject(kubeAuthProxyCertificateInjectable, clusterUrl.hostname);
+    const clusterApiUrl = di.inject(clusterApiUrlInjectable, cluster);
+    const createKubeAuthProxy = di.inject(createKubeAuthProxyInjectable, cluster);
 
     let kubeAuthProxy: KubeAuthProxy | undefined = undefined;
     let apiTarget: ServerOptions | undefined = undefined;
```

Comment:
After this change, we use validated kubeconfig. Previously we didn't (?)?

Reply:
which change in particular?

Reply:
`const clusterApiUrl = di.inject(clusterApiUrlInjectable, cluster);` vs `const clusterUrl = new URL(cluster.apiUrl.get());`